index.html

```diff
@@ -228,10 +228,10 @@ <h2><code>PointerEvent</code> interface</h2>
                 <dl data-dfn-for="PointerEvent" data-link-for="PointerEvent">
                     <dt><dfn>pointerId</dfn></dt>
                         <dd>
-                            <p>A unique identifier for the pointer causing the event. This identifier MUST be unique from all other <a data-lt="active pointer">active pointers</a> in the <a>top-level browsing context</a> (as defined by [[HTML]]) at the time. The <code>pointerId</code> value of <code>-1</code> is reserved to indicate events that were generated by something other than a pointing device. The user agent MAY recycle previously retired values for <code>pointerId</code> from previous active pointers, if necessary.</p>
+                            <p>A unique identifier for the pointer causing the event. This identifier MUST be unique from all other <a data-lt="active pointer">active pointers</a> in the <a>top-level browsing context</a> (as defined by [[HTML]]) at the time, and the identifier MUST NOT be influenced by any other top-level browsing context. The <code>pointerId</code> value of <code>-1</code> is reserved to indicate events that were generated by something other than a pointing device. The user agent MAY recycle previously retired values for <code>pointerId</code> from previous active pointers, or it MAY always reuse the same <code>pointerId</code> for a particular pointing device for the lifetime of the page. However, in the latter case, the <code>pointerId</code> MUST be randomized, to minimize the chance of fingerprinting and tracking across different pages.</p>
```

Comment:
Does it look a bit less hand-wavy?

> ...MUST NOT be influenced by _existing pointers in_ any other... (emphasis marks the new text)

Not sure if this would leave something important out!